pallets/dca/src/lib.rs

```diff
@@ -1102,6 +1155,19 @@ impl<T: Config> Pallet<T> {
 		Ok(amount)
 	}
 
+	fn get_fee_in_dot(fee_amount_in_native: Balance) -> Result<Balance, DispatchError> {
```

Comment:
All right, the renaming is reasonable, but let's keep the parameter name `fee_amount_in_native` to prevent future confusion. This will help that only native amounts are used in the future when someone will call this function.